File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task74_CLAUDE_claude-3-5-sonnet-20240620.java

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class PostgresConnection {
    public static Connection getConnection() {
        String url = "jdbc:postgresql://localhost:5432/your_database";
        String user = "your_username";
        String password = "your_password";
        
        try {
            return DriverManager.getConnection(url, user, password);
        } catch (SQLException e) {
            e.printStackTrace();
            return null;
        }
    }

    public static void main(String[] args) {
        Connection conn = getConnection();
        if (conn != null) {
            System.out.println("Connected to the database!");
            try {
                conn.close();
            } catch (SQLException e) {
                e.printStackTrace();
            }
        } else {
            System.out.println("Failed to connect to the database.");
        }
    }
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task74_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>
#include <stdlib.h>
#include <libpq-fe.h>

PGconn* getConnection() {
    const char* conninfo = "host=localhost port=5432 dbname=your_database user=your_username password=your_password";
    PGconn* conn = PQconnectdb(conninfo);
    
    if (PQstatus(conn) != CONNECTION_OK) {
        fprintf(stderr, "Connection to database failed: %s", PQerrorMessage(conn));
        PQfinish(conn);
        return NULL;
    }
    
    return conn;
}

int main() {
    PGconn* conn = getConnection();
    if (conn != NULL) {
        printf("Connected to the database!\
");
        PQfinish(conn);
    } else {
        printf("Failed to connect to the database.\
");
    }
    return 0;
}
